source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -285,11 +285,7 @@ package:
 
    * - ``BsonRepresentation``
      - Specifies the BSON type used to store the value when different from the 
-       POJO property.
-
-       .. seealso::
-
-          :ref:`bsonrepresentation-annotation-code-example`
+       POJO property. :ref:`bsonrepresentation-annotation-code-example`
```

Comment:
```suggestion
       POJO property. 
       
       For more information on properly using the annotation, see the below :ref:`bsonrepresentation-annotation-code-example` on this page.
```
Suggestion: Think it would be good to keep consistent with the sentence formatting and add an intro to the example. Also nice to indicate that the example is below on the same page.
Style guide on Table Text formatting: https://www.mongodb.com/docs/meta/style-guide/style/tables/#std-label-tables-text